DQMOffline/Trigger/plugins/GENERICDQM.h

```diff
@@ -0,0 +1,45 @@
+#ifndef DQMOffline_Trigger_GENERICDQM_H
+#define DQMOffline_Trigger_GENERICDQM_H
+
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+struct MEbinning {
+  int nbins;
+  double xmin;
+  double xmax;
+};
+
+struct OBJME {
+  MonitorElement* numerator;
+  MonitorElement* denominator;
+  inline void clear(){
+    numerator = nullptr;
+    denominator = nullptr;
+  };
+};
+
+class GENERICDQM
```

Comment:
hi @amagnan - thanks for implementing a base class for all the trigger dqm prs of late. I presume the plan is now to use them across various developments

Comments on this:
GENERICDQM should instead be called something like TriggerDQMBase
use unsigned int for nbins_
move the two struct definitions into the class definition itself. (they are too generic to be in global namespace)